frontend/src/components/Wheel/index.js

```diff
@@ -1,12 +1,39 @@
 import { useEffect, useState } from "react";
-import SpinButton from "./button";
 import styles from "./index.module.css";
+import "./index.css";
 
 const Wheel = () => {
   const [wheelInfo, setWheelInfo] = useState({});
+  const [rotate, setRotate] = useState("circle");
+
+  async function startRotate() {
```

Comment:
Why is this function `async`?

Reply:
By this time we didn't know why it should be async and we removed it. Then we put a fetch to the back-end to give the prize the user as won and determined that the fetch should have an await (to not stop spinning before a prize is awarded) and therefore the startRotate actually ended up async again.